integration_tests/tests/appointments/logEnforcement.cy.ts

```diff
@@ -0,0 +1,56 @@
+//  Feature: Update log compliance
```

Comment:
Is it worth adding a integration test in logCompliance for navigating to the enforcement page?